utils.py

```diff
@@ -3781,6 +3786,7 @@ def _getDatasetBlocksFraction(url, dataset, complete='y', group=None, vetoes=Non
         vetoes = ['MSS','Buffer','Export']
 
     # VK TODO: replace with call to get list of blocks for a given dataset
+    # JRV: need only the blocks with valid files.
```

Comment:
I think DBS provide a way to get list of blocks with valid files, please consult with Yuyi